src/System.IO.Compression/src/System/IO/Compression/ZLibException.Serialization.cs

```diff
@@ -0,0 +1,35 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.Serialization;
+using System.IO.Compression;
+
+namespace System.IO.Compression
+{
+    [Serializable]
+    internal partial class ZLibException : IOException, ISerializable
```

Comment:
I don't think that would really help. We'd still need to provide the real implementation. It works for an attribute as there's no code.